app/models/container_group.rb

```diff
@@ -75,4 +75,18 @@ def perf_rollup_parents(interval_name = nil)
       ([container_project, container_replicator] + container_services).compact
     end
   end
+
+  def disconnect_inv
+    _log.info "Disconnecting Container group [#{name}] id [#{id}] from EMS [#{ext_management_system.name}]" \
+    "id [#{ext_management_system.id}] "
+    self.old_ems_id = ems_id
+    self.ext_management_system = nil
+    self.container_node_id = nil
+    self.container_services = []
+    self.container_replicator_id = nil
+    self.container_build_pod_id = nil
```

Comment:
@simon3z So to make sure you want me to add history keeping of:
nodes, services, replicators, build pods.

shouldn't we only keep what we need for chargeback?